### PR TITLE
Update free-programming-books-langs.md

### PR DESCRIPTION
**Pull request by adarsh-jha-dev**
## What does this PR do?
I have added a couple of free eBooks in the C++ section. 

### Description
The books which i have added are : 

- Practical C++ Programming by Steve Oualline (https://www.mimuw.edu.pl/~mrp/cpp/SecretCPP/O'Reilly%20-%20Practical%20C++%20Programming.pdf) - Steve Oualline (PDF)
- Lecture Notes On Object Oriented Programming Using C++ (https://www.cet.edu.in/noticefiles/285_OOPS%20lecture%20notes%20Complete.pdf) - Dr. Subasish Mohapatra (PDF)

### Why is this valuable (or not)?
I have read those eBooks and it seems beneficial for beginners.

### How do we know it's really free?
I have cross-verified by checking the links and reading those books

### For book lists, is it a book? For course lists, is it a course? etc.
These are books.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [*] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [*] Include author(s) and platform where appropriate.
- [*] Put lists in alphabetical order, correct spacing.
- [*] Add needed indications (PDF, access notes, under construction).
- [*] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by 3R1Dev** [comment]
Both of the added links are to sub directories from schools, the links appear to be intended for use inside class
or for students. While they can be found via web scrapping methods, the authority of sharing those links is questionable.

It is possible the school has permission to share them but unless you can know with 100% certainty it is better to not add them.

**Comment by adarsh-jha-dev** [comment]
These books are publicly available on the internet so it's evident that the school doesn't care about this. Also where i got these books from, there is nothing mentioned about the School and all. Even after this if you want i can remove these books and replace with some other ones. Please do let me know about this.

**Comment by adarsh-jha-dev** [comment]
I have removed that particular book which was creating problem and have replaced it with another free eBook.

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -536,9 +536,9 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Joint Strike Fighter, C++ Coding Standards](http://www.stroustrup.com/JSF-AV-rules.pdf) - Bjarne Stroustrup (PDF)
 * [LearnCpp.com](https://www.learncpp.com) (HTML)
 * [Learning C++ eBook](https://riptutorial.com/Download/cplusplus.pdf) - Compiled from StackOverflow Documentation (PDF)
-* [Lecture Notes On Object Oriented Programming Using C++](https://www.cet.edu.in/noticefiles/285_OOPS%20lecture%20notes%20Complete.pdf) - Dr. Subasish Mohapatra (PDF)
 * [Matters Computational: Ideas, Algorithms, Source Code](http://www.jjj.de/fxt/fxtbook.pdf) - Jorg Arndt (PDF)
 * [More C++ Idioms](https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms) - Sumant Tambe, et al. (WikiBooks)
+* [Object Oriented Programming using 'C++'](https://www.ddegjust.ac.in/studymaterial/msc-cs/ms-17.pdf) - Pooja Chawla (PDF)
```
Comment (sohansai) [inline books/free-programming-books-langs.md:541]: The link you provided isn't a book; it appears to be a study material for a university course.

**Comment by adarsh-jha-dev** [comment]
I have resolved all the issues with the newly added books. I have removed those problematic ones and replaced it with some good ones.

**Comment by adarsh-jha-dev** [comment]
I have resolved all the issues by replacing those books, i request you to please merge these changes after verifying the same.

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -516,6 +516,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [C++ Core Guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md) - `edt.:` Bjarne Stroustrup, Herb Sutter
 * [C++ For Programmers](https://tfetimes.com/wp-content/uploads/2015/04/c-for-c-programmers.pdf) - JT Kalnay (PDF)
 * [C++ GUI Programming With Qt 3](https://ptgmedia.pearsoncmg.com/images/0131240722/downloads/blanchette_book.pdf) - Jasmin Blanchette, Mark Summerfield (PDF)
+* [C++ In Action: Industrial Strength Programming Techniques](http://www.worldcolleges.info/sites/default/files/C%2B%2B_In_Action.pdf) - Bartosz Milewski (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:519]: It's published by Pearson, they don't even let the author distribute it. Also it's >25 years old. We won't bother with it..

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -534,11 +535,13 @@ Books on general-purpose programming that don't focus on a specific language are
 * [How To Think Like a Computer Scientist: C++ Version](http://greenteapress.com/thinkcpp/index.html) - Allen B. Downey
 * [Introduction to Design Patterns in C++ with Qt 4](http://ptgmedia.pearsoncmg.com/images/9780131879058/downloads/0131879057_Ezust_book.pdf) - Alan Ezust, Paul Ezust (PDF)
 * [Joint Strike Fighter, C++ Coding Standards](http://www.stroustrup.com/JSF-AV-rules.pdf) - Bjarne Stroustrup (PDF)
+* [Learn C++ Programming Language](http://www.tutorialspoint.com/cplusplus/cpp_tutorial.pdf) - Tutorials Point (PDF)
 * [LearnCpp.com](https://www.learncpp.com) (HTML)
 * [Learning C++ eBook](https://riptutorial.com/Download/cplusplus.pdf) - Compiled from StackOverflow Documentation (PDF)
 * [Matters Computational: Ideas, Algorithms, Source Code](http://www.jjj.de/fxt/fxtbook.pdf) - Jorg Arndt (PDF)
 * [More C++ Idioms](https://en.wikibooks.org/wiki/More_C%2B%2B_Idioms) - Sumant Tambe, et al. (WikiBooks)
 * [Open Data Structures (In C++)](http://opendatastructures.org/ods-cpp.pdf) - Pat Morin (PDF)
+* [Practical C++ Programming by Steve Oualline](https://www.mimuw.edu.pl/~mrp/cpp/SecretCPP/O'Reilly%20-%20Practical%20C++%20Programming.pdf) - Steve Oualline (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:544]: This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.

**Review by eshellman — CHANGES_REQUESTED** [review]
the tutorialspoint book is ok, remove the others.

**Comment by adarsh-jha-dev** [comment]
Other than the tutorials point's one, i have removed all the unauthorized distributed books.

**Pull request merged**